projects/cashmere/src/lib/sidenav/sidenav-link/sidenav-link.component.scss

```diff
@@ -3,30 +3,81 @@
 a {
     color: $white;
     font-size: 16px;
-    line-height: 1.19;
+    line-height: 19px;
     width: 100%;
     display: block;
-    padding: 19px 15px 18px 28px;
 
-    &:hover {
-        background-color: $dark-blue-hover;
-        text-decoration: none;
+    .top-most-link {
+        padding: 15px 15px 14px 26px;
+
+        &:hover {
+            background-color: $wcf-blue;
+            text-decoration: none;
+        }
+        .toggle-button:hover {
+            background-color: $wcf-blue;
+        }
     }
 
     &:active {
         background-color: $wcf-blue;
     }
 
     hc-icon {
-        margin-right: 15px;
+        margin-right: 26px;
         vertical-align: middle;
     }
 
     .children {
-        margin-top: 20px;
+
+        .top-most-link {
+            height: 0;
+            opacity: 0;
+            overflow: hidden;
+            padding-top: 0;
+            padding-bottom: 0;
+            padding-left: 44px;
+            transition: 0.3s ease;
+        }
+
+        &.shown {
+            .top-most-link {
+                height: 50px;
```

Comment:
Did this do the trick then or are some of the sub children still not showing?

Reply:
All sub children are now showing! It's great.